src/pages/VolunteersJobListingFS.page

```diff
@@ -293,6 +293,9 @@
                     return false;" 
                 immediate="false" status="statusSignUp" styleClass="cssButton btn" />              
             <apex:commandButton value="{!$Label.labelButtonCancel}" immediate="true" onclick="j$('#dlgSignUp').dialog('close');" action="{!CancelSignUp}" styleClass="cssButton btn" />            
+            <div id="captcha" style="margin-top:5px;">
```

Comment:
On failure do we want to try and keep the modal open?

Reply:
For sure. The modal is closing now because I stopped setting the expected return string. Fixed in the upcoming commit.